src/app/modules/ix-forms/components/ix-dynamic-form/ix-dynamic-form.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
+import { IxDynamicFormComponent } from 'app/modules/ix-forms/components/ix-dynamic-form/ix-dynamic-form.component';
+
+describe('IxDynamicFormComponent', () => {
+  let spectator: Spectator<IxDynamicFormComponent>;
+  const createComponent = createComponentFactory({
+    component: IxDynamicFormComponent,
+  });
+
+  beforeEach(() => {
+    spectator = createComponent();
+  });
+
+  describe('Control Emit', () => {
+    it('addControlNext()', () => {
```

Comment:
Same here, it's better to do `MockComponent(IxDynamicFormItem)` and then check what's being passed in.
```
const dynamicItem = spectator.query(IxDynamicFormItem);
expect(dynamicItem.schema).toEqual(...);

// In another case
dynamicItem.addListItem.emit(...)
// expect that parent listener has been called.
```